data/coalition/coalition news

```diff
@@ -0,0 +1,177 @@
+			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
+			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
+			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`
+			"An agent is trying to calm down a small crowd outside a local bar. After some minutes the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."
+
+news "heliarch candidate"
+	location
+		government "Heliarch"
+	name
+		word
+			"Heliarch candidate"
+	message
+		word
+			`"Laws, history, theory, too much of it I have read, and still behind I am..."`
+			`"As part of my trials, read the works of elder Elirom, I had to. Exceeded my expectations, his books have."`
+			`"To prove our dedication and show our potential, the trials we candidates go through are. Though difficult, we're sure to be apt to serve once passed, we have."`
+
+news "coalition ad"
+	location
+		government "Coalition"
+	name
+		word
+			"Advertisement"
+	message
+		word
+			"You see a commercial where a Saryd sits down after a workout, and as the camera spins it switches to a Kimek studying, and then to an Arach watching sports while cheering. It seems to be an ad for a brand of foldable chairs."
+			"A commercial for house Debdo plays on a local monitor, with various scenes of tourist spots being accompanied by an upbeat, serene jingle."
+			"An ad shows various graphs of what you guess are stock values. An Arach leg swipes the view and the graphs begin fluctuating a lot more to one side. At the end the logo of house Plumtab flashes on the screen."
+			"An ad for an energy drink is on display on a local shop. A Kimek is down on the ground in the middle of the streets, struggling to move, as a crowd gathers around them. When they get the drink, they jump up and bolt past the crowd, and then proceed to climb up a skyscraper. At the end a disclaimer advises against Saryds consuming the drink."
+
+news "arach tourist"
+	location
+		government "Coalition"
+		attributes "tourism"
+		not attributes "arach" "station"
+	name
+		word
+			"Arach tourist"
+	message
+		word
+			`"Ever since another child I had, extended my vacation period and increased my holiday pay, the government has."`
+			`"Much better than I expected, the food here is. Still, beat longcow steak, nothing here does."`
+			`"That the streets here were too narrow for us Arachi, I had heard, but no problem so far I've had."`
+
+news "kimek tourist"
+	location
+		government "Coalition"
+		attributes "tourism"
+		not attributes "kimek" "station"
+	name
+		word
+			"Kimek tourist"
+	message
+		word
+			`"Come here with a caravan, I have. Cheaper it was, but stick together we must, and too tight the schedule they set up is, barely appreciate the sights, I can!"`
```

Comment:
```suggestion
			`"Come here with a caravan, I have. Cheaper it was, but stick together we must, and too tight the schedule they set up is. Barely appreciate the sights, I can!"`
```
It's grammatically incorrect, but since it's dialogue it could slide if you wanted. Your choice.